package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "body-parser": "^1.19.0",
+    "cloudinary": "^1.22.0",
```

Comment:
Are you connecting to cloudinary to get your photos? There's a simple way to get a JSON of all the photo data and stick it in a raw data js file. Up to you, whatever you're more comfortable with, its just in terms of simplicity and time contraints.